src/initial_conditions/initial_conditions.jl

```diff
@@ -160,6 +160,31 @@ function (initial_condition::DecayingProfile)(params)
     return local_state
 end
 
+"""
+    DYAMONDSummer(; perturb = true)
+
+An `InitialCondition` with a decaying temperature profile, and with an optional
+perturbation to the temperature.
```

Comment:
This is not updated. There's comments/explanation on the fact that this initial condition relies on calling another function 
in another function in another file.

Reply:
Fixed.